spring-cloud-gateway-webflux/src/test/java/org/springframework/cloud/gateway/webflux/ProductionConfigurationTests.java

```diff
@@ -204,11 +204,12 @@ public void headers() throws Exception {
 						Map.class)
 				.getBody();
 		assertThat(headers).doesNotContainKey("foo").doesNotContainKey("hello")
-				.containsKeys("bar", "abc");
+				.containsKeys("bar", "abc", "Authorization");
```

Comment:
This is not part of spring-cloud-gateway-server and should be removed from this enhancement since this module doesn't know anything about server properties.